BackendTaskPilot/BackendTaskPilot.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
Ensure that the version aligns with the version in our other services and libraries or use latest `net8.0` if none exist